vite.config.shared.ts

```diff
@@ -149,10 +149,11 @@ export const getApiProxies = (pathToRoot: string = '../../../.') => {
   // Add additional regions as they become available
   const availableRegions = ['au', 'eu', 'us']
   const regionalProxies = {}
+  const regionalV2Proxies = {}
   // Build the regional API proxies
   for (const region of availableRegions) {
     // @ts-ignore
-    regionalProxies[`^/${ region }/kong-api/konnect-api`] = {
+    regionalProxies[`^/${ region }/kong-api`] = {
```

Comment:
Some formatting seems off in this file; can you run `pnpm run lint:fix` from the root and push up any changes, if there are any?

Reply:
no file changes after command